Common/src/CConfig.cpp

```diff
@@ -1223,6 +1223,10 @@ void CConfig::SetConfig_Options() {
   addDoubleOption("PRANDTL_LAM", Prandtl_Lam, 0.72);
   /*!\brief PRANDTL_TURB \n DESCRIPTION: Turbulent Prandtl number (0.9 (air), only for compressible flows) \n DEFAULT 0.90 \ingroup Config*/
   addDoubleOption("PRANDTL_TURB", Prandtl_Turb, 0.90);
+  /*!\brief WALLMODELKAPPA \n DESCRIPTION: von Karman constant used for the wall model \n DEFAULT 0.41 \ingroup Config*/
+  addDoubleOption("WALLMODELKAPPA", wallModelKappa, 0.41);
+  /*!\brief WALLMODELB \n DESCRIPTION: constant B used for the wall model \n DEFAULT 5.0 \ingroup Config*/
+  addDoubleOption("WALLMODELKAPPA", wallModelKappa, 0.41);
```

Comment:
:+1: for putting it in the config (you missed B)
Is it common to specify a specific von Karman constant for wall models? Or should it be simply called VONKARMANCONST?

Reply:
Ha, guess I pushed and then called it a day.

I think the value is pretty fixed, usually 0.40 or 0.41, for every turbulence model that uses it. But I  don't want to give a user the impression that you can change every occurrence of this constant in all the turbulence models in the code.  So for now, it's only for the wall model, which is reflected in the name.

Reply:
Makes sense